### DISCUSSION TITLE
Spacy version 3 model training using training loop (without CLI)

### DISCUSSION THREAD

**Discussion by mandar-avhad** [explosion/spaCy#8771] [Help: Coding & Implementations]
When trying to load the spacy pretrained en_core model and train it on new data, getting the either of these errors.
optimizer = nlp.resume_training()
ValueError: Cannot get dimension 'nO' for model 'sparse_linear': value unset

optimizer = nlp.initialize(lambda: examples)
ValueError: [E955] Can't find table(s) lexeme_norm for language 'en' in spacy-lookups-data. Make sure you have the package installed or provide your own lookup tables if no default lookups are available for your language.

Please help me with the training part in version 3, without the CLI method.

**Comment by mandar-avhad** [comment]
@polm Help please?

**Reply by polm** [reply]
1. Do not @ maintainers. Especially do not @ maintainers after less than 24 hours. We try to help everyone but with no guarantees.
2. This is a duplicate of https://github.com/explosion/spaCy/discussions/8764. Do not open multiple threads for the same topic.

I will be locking this thread; I'll follow up in the other one.
